src/indexer/utils/substrate/index.ts

```diff
@@ -137,6 +139,9 @@ export async function saveDeposit(
       timestamp: timestamp,
       destination: `0x${depositData.substring(2).slice(128, depositData.length - 1)}`,
     } as Pick<DecodedDepositLog, "depositNonce" | "sender" | "amount" | "destination" | "resourceID" | "toDomainId" | "fromDomainId" | "timestamp">
+
+    await accountRepository.insertAccount({ id: sender, addressStatus: "" })
```

Comment:
It is posible that this account already exist

Reply:
I have changed that method to use `upsert` instead of create, since it was giving errors regarding unique constraint